indico/modules/categories/templates/management/move_category_contents.html

```diff
@@ -1,16 +1,48 @@
 <div class="action-box">
     <div class="section">
-        <span class="icon icon-transmission"></span>
         <div class="text">
-            <div class="label">{% trans %}Selected categories to be moved{% endtrans %}:</div>
-            <div>{{ category.title }}</div>
+            <div class="label">
+                {% if categories %}
+                    {% trans count=categories|count, category=(categories|first).title%}
+                        Select where you want to move "{{ category }}".
+                    {% pluralize %}
+                        Select where you want to move the following categories:
+                    {% endtrans %}
+                {% endif %}
+            </div>
+            {% if categories|count > 1 %}
+            <ul>
+                {% for category in categories %}
+                <li>{{ category.title }}</li>
+                {% endfor %}
+            </ul>
+            {% endif %}
         </div>
     </div>
 </div>
 <div id="categorynav-holder"></div>
 <script>
-    $('#categorynav-holder').categorynavigator({
-        categoryId: {{ category.parent.id if not category.is_root else category.id }},
-        selectLeafOnly: true
+    var $categorynav = $('#categorynav-holder');
```

Comment:
creates a global; wrap in 

    (function() {
        'use strict';
        
    })();